src/utilities/command-utils.test.ts

```diff
@@ -0,0 +1,188 @@
+import { TestUtils } from "../tests/test-utils";
+import { CommandUtils } from "./command-utils";
+import program, { Command } from "commander";
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("CommandUtils", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region Setup
+    // -----------------------------------------------------------------------------------------
+
+    const seedCommandWithSubcommand = (
+        subcommandName: string
+    ): program.Command =>
+        new Command("parent").addCommand(new Command(subcommandName));
+
+    // #endregion Setup
+
+    // -----------------------------------------------------------------------------------------
+    // #region equalsByName
+    // -----------------------------------------------------------------------------------------
+
+    describe("equalsByName", () => {});
+
+    // #endregion equalsByName
+
+    // -----------------------------------------------------------------------------------------
+    // #region exists
+    // -----------------------------------------------------------------------------------------
+
+    describe("exists", () => {});
+
+    // #endregion exists
+
+    // -----------------------------------------------------------------------------------------
+    // #region get
+    // -----------------------------------------------------------------------------------------
+
+    describe("get", () => {
+        test.each([undefined, null, "", " "])(
+            "when called with %p, it returns undefined",
```

Comment:
Minor, but any reason `%p` (pretty format) is used here instead of something just like `$name`?

Reply:
Responded in that other code review but I'll add it here for anyone else wondering - I believe the parameter replacement only works with the template string overload of `test.each`, not the `test.each([table])` version.

[`test.each([table])`](https://jestjs.io/docs/en/api#1-testeachtablename-fn-timeout)

[`test.each(templated string)`](https://jestjs.io/docs/en/api#2--testeachtablename-fn-timeout-)

Reply:
ah! overlooked that - thanks for clearing that up